### DISCUSSION TITLE
Sharing a Project is Difficult

### DISCUSSION THREAD

**Discussion by chances** [community/community#70626] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

Please consider adding a button that allows me to quickly Share a project.

As it is now, I must copy-and-paste the URL, which I'm not sure is even a permalink.

([Referred](https://github.com/orgs/community/discussions/16996#discussioncomment-7067804) here by @ghostinhershell.)

**Comment by ghostinhershell** [comment]
Hey @chances, thanks so much for sharing your feedback here!

**Comment by hans-djalali** [comment]
@chances
For the moment, there isn't a simpler, built-in solution for sharing projects directly from GitHub. However, your feedback about adding a share button is valuable and could be a great suggestion to raise directly with GitHub. They often update their platform based on user feedback, so sharing your idea could potentially lead to new features that make project sharing easier in the future. I really would like to have this feature as well :)
